doc/contributing.rst

```diff
@@ -115,6 +115,13 @@ list of these, see ``testament/categories.nim``, at the bottom.
 
   ./koch tests c lib
 
+To run a single test:
+
+::
+
+  ./koch tests c <category>/<name>
```

Comment:
@FedericoCeratto isn't that a typo?
bug, this should be `./koch tests r <category>/<name>` (r not c) ?

Reply:
fixed in https://github.com/nim-lang/Nim/pull/10571